xbmc/application/Application.cpp

```diff
@@ -2394,8 +2394,7 @@ bool CApplication::PlayFile(CFileItem item, const std::string& player, bool bRes
       if (videoInfoTagPath.find("removable://") == 0 || item.IsVideoDb())
         path = videoInfoTagPath;
 
-      if (!item.HasVideoInfoTag())
-        dbs.LoadVideoInfo(path, *item.GetVideoInfoTag());
+      dbs.LoadVideoInfo(path, *item.GetVideoInfoTag());
```

Comment:
Maybe add a comment otherwise someone will refactor this again without taking it into conIderation

Reply:
Good idea.